src/web/app/components/copy-session-modal/copy-session-modal.component.spec.ts

```diff
@@ -30,4 +40,65 @@ describe('CopySessionModalComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should snap with default fields', () => {
+    expect(fixture).toMatchSnapshot();
+  });
+
+  const feedbackSessionToCopy: FeedbackSession = {
+    courseId: 'Test01',
+    timeZone: 'Asia/Singapore',
+    feedbackSessionName: 'Test session',
+    instructions: 'Answer all',
+    submissionStartTimestamp: 1555232400,
+    submissionEndTimestamp: 1555332400,
+    gracePeriod: 0,
+    sessionVisibleSetting: SessionVisibleSetting.AT_OPEN,
+    responseVisibleSetting: ResponseVisibleSetting.AT_VISIBLE,
+    submissionStatus: FeedbackSessionSubmissionStatus.OPEN,
+    publishStatus: FeedbackSessionPublishStatus.NOT_PUBLISHED,
+    isClosingEmailEnabled: true,
+    isPublishedEmailEnabled: true,
+    createdAtTimestamp: 1554967204,
+  };
+
+  const courseSessionIn: Course = {
+    courseId: 'Test01',
+    courseName: 'Sample Course 101',
+    creationDate: '13 Mar 2019',
+    deletionDate: 'Not Applicable',
+    timeZone: 'UTC',
+  };
+
+  const courseCopyTo: Course = {
+    courseId: 'Test02',
+    courseName: 'Sample Course 202',
+    creationDate: '14 Mar 2019',
+    deletionDate: 'Not Applicable',
+    timeZone: 'UTC',
+  };
+
+  it('should snap with some session and courses candidates', () => {
+    component.newFeedbackSessionName = feedbackSessionToCopy.feedbackSessionName;
+    component.courseCandidates = [courseSessionIn, courseCopyTo];
+    component.sessionToCopyCourseId = courseSessionIn.courseId;
+    fixture.detectChanges();
+
+    expect(fixture).toMatchSnapshot();
+  });
+
+  it('should snap with course to copy to is set', () => {
+    component.newFeedbackSessionName = feedbackSessionToCopy.feedbackSessionName;
+    component.courseCandidates = [courseSessionIn, courseCopyTo];
+    component.sessionToCopyCourseId = courseSessionIn.courseId;
+    fixture.detectChanges();
+
+    const options: DebugElement[] = fixture.debugElement.queryAll(By.css('input[type="radio"]'));
+    const secondOption: any = options[1];
+    secondOption.triggerEventHandler('change', { target: secondOption.nativeElement });
+    fixture.detectChanges();
+
+    expect(fixture).toMatchSnapshot();
```

Comment:
What is the snapshot trying to verify?

It is not a good practice to snapshot everything when you trigger some events.